ios/Video/RCTVideo.m

```diff
@@ -967,7 +991,8 @@ - (void)applyModifiers
     [_player setVolume:_volume];
     [_player setMuted:NO];
   }
-  
+
+  [self setPitchAlgorithm:_pitchAlgorithm];
```

Comment:
@cobarx Hey! I'm not sure if we should call setPitchAlgorithm in applyModifiers or not? I'm not ObjC dev.

Reply:
I'm not sure on this... it has been awhile and I can't remember why we have to re-apply the modifiers. Short answer is as long as the call to set it is near instantaneous it won't hurt anything.